core/tests/protractor_desktop/playVoiceovers.js

```diff
@@ -0,0 +1,109 @@
+  '../protractor_utils/CreatorDashboardPage.js');
+var ExplorationPlayerPage = require(
+  '../protractor_utils/ExplorationPlayerPage.js');
+var LibraryPage = require('../protractor_utils/LibraryPage.js');
+
+describe('Voiceover player', function() {
+  var creatorDashboardPage = null;
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationEditorTranslationTab = null;
+  var explorationEditorSettingsTab = null;
+  var explorationPlayerPage = null;
+  var libraryPage = null;
+
+  beforeAll(async function() {
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    explorationEditorTranslationTab = (
+      explorationEditorPage.getTranslationTab());
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    libraryPage = new LibraryPage.LibraryPage();
+
+    await users.createUser('testVoiceovers@voiceovers.com',
+      'testVoiceovers');
+    await users.login('testVoiceovers@voiceovers.com');
+    await workflow.createExploration();
+    await explorationEditorMainTab.exitTutorial();
+    await explorationEditorMainTab.setStateName('First');
+    await explorationEditorMainTab.setContent(await forms.toRichText(
+      'This is the first card.'));
+    await explorationEditorMainTab.setInteraction('EndExploration');
+    await explorationEditorPage.navigateToTranslationTab();
+    await explorationEditorTranslationTab.exitTutorial();
+    await explorationEditorTranslationTab.uploadAudioFileForLanguage(
+      'Hindi', '../data/cafe.mp3');
+    await explorationEditorTranslationTab.uploadAudioFileForLanguage(
+      'Arabic', '../data/ambient-noise.mp3');
+    await explorationEditorPage.saveChanges('Added voiceovers');
+    await explorationEditorPage.navigateToSettingsTab();
+    await explorationEditorSettingsTab.setTitle('voiceoverPlayerTest');
+    await explorationEditorSettingsTab.setCategory('Languages');
+    await explorationEditorSettingsTab.setLanguage('English');
+    await explorationEditorSettingsTab.setObjective(
+      'Testing if voiceovers work');
+    await explorationEditorPage.saveChanges('Done.');
+    await workflow.publishExploration();
+    await users.logout();
+  });
+
+  beforeEach(async function() {
+    await users.login('testVoiceovers@voiceovers.com');
+  });
+
+  it('should play and pause voiceovers', async function() {
+    await libraryPage.get();
+    await libraryPage.playExploration('voiceoverPlayerTest');
+    await explorationPlayerPage.expandAudioBar();
+    await explorationPlayerPage.pressPlayButton();
+    await explorationPlayerPage.expectAudioToBePlaying();
+    await explorationPlayerPage.pressPauseButton();
+    await explorationPlayerPage.expectAudioToBePaused();
```

Comment:
Hi @krishnarao22, this e2e test is failing repeatedly on circleCI and seems like flaky, can you help to investigate this? 

<details>
<summary>Error stack trace</summary>
<pre>
1) Voiceover player should play and pause voiceovers
  Message:
    Expected false to be truthy.
  Stack:
    Error: Failed expectation
        at ExplorationPlayerPage.expectAudioToBePaused (/home/circleci/oppia/core/tests/protractor_utils/ExplorationPlayerPage.js:112:42)
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (internal/process/task_queues.js:97:5)
        at UserContext.<anonymous> (/home/circleci/oppia/core/tests/protractor_desktop/playVoiceovers.js:89:5)
</pre>
</details>

Few improvements in the code:

1. Please select a language before playing audio (Maybe it's playing auto-generated audio not sure!)
2. Add more checks inside expectAudioToBePlaying like audio progress bar is visible (maybe?)

Can you please take a closer look and see if you can reproduce this locally? I'll connect with you on hangout for a quick discussion!